programs/github

```diff
@@ -16,10 +16,12 @@ local subcommands = {
 		}
 	},
 	auth = {
-		usage = "github auth <user> [<api token> | -d ]",
+		usage = "github auth <user> [<api token> | -r | -d]",
```

Comment:
With the recommended changes wouldn't this be `github Auth <user> (<api token> [-d | - - set-default]) `